### PR TITLE
Feature aha docs

### PR DESCRIPTION
**Pull request by vEpiphyte**
* add aha docs
* Add perms on aha csr generation
* add synapse.tools.aha.list and synapse.tools.aha.easycert for interacting with aha cells
* move lots of aha telepath api login into the cell itself
* add support for csrs with host sans
* add the celll ``aha:svcinfo`` configuration 
* update aha tests to account for cell mirroring
* update autodoc for cell to extract informaition about ``object`` types for documentation.
* log when an Aha service is torn down and unable to clear the online state of a service
* fix logging order of operations issue in ``Cell.initFromArgv``
* fix a teardown issue in cmdr causing the teleenv not to be torn down

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -258,5 +259,5 @@ def setup(app):
     app.connect('builder-inited', run_apidoc)
     app.connect('builder-inited', run_modeldoc)
     app.connect('builder-inited', run_confdocs)
-    app.connect('builder-inited', convert_ipynb)
+    # app.connect('builder-inited', convert_ipynb)
```
Comment (vEpiphyte) [inline docs/conf.py:262]: remove this

File: docs/synapse/devguides/devops_aha.rst [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+      - - ssl://bob@127.0.0.1:8081/?hostname=aha.demo.net
+
+Now Aha can be used to connect to the Cortex::
+
+    python -m synapse.tools.cmdr "aha://bob@core02.demo.net/"
+
+TODO
+----
+
+SVCINFO notes
```
Comment (vEpiphyte) [inline docs/synapse/devguides/devops_aha.rst:328]: Fill this out

File: synapse/lib/aha.py [hunk 3]
```diff
@@ -169,7 +140,11 @@ def _nameAndNetwork(self, name, network):
 
         if network is None:
             svcfull = name
-            svcname, svcnetw = name.split('.', 1)
+            try:
+                svcname, svcnetw = name.split('.', 1)
+            except ValueError:
+                raise s_exc.BadArg(name=name, arg='name',
+                                   mesg='Name must contain at least one "."')
```
Comment (vEpiphyte) [inline synapse/lib/aha.py:147]: svcname

File: docs/synapse/devguides/devops_aha.rst [hunk 4]
```diff
@@ -0,0 +1,428 @@
+.. _devops-aha:
+
+.. highlight:: none
+
+Aha Service Discovery
+=====================
+
+The Aha service discovery is a native Synapse tool and protocol for doing
+service discovery when multiple Synapse services are being deployed in a
```
Comment (invisig0th) [inline docs/synapse/devguides/devops_aha.rst:9]: `an environment`

File: docs/synapse/devguides/devops_aha.rst [hunk 5]
```diff
@@ -0,0 +1,428 @@
...
+environment. This page covers what how Aha works, what services the AhaCell
```
Comment (invisig0th) [inline docs/synapse/devguides/devops_aha.rst:10]: `what how Aha works` ? :)

**Review by invisig0th — CHANGES_REQUESTED** [review]

**Review by invisig0th — APPROVED** [review]

**Pull request merged**